### PR TITLE
My Portfolio

### PR DESCRIPTION
**Pull request by zakiyousufi**
We created a contact form that
The name field accepts up to 30 chars.
The email field accepts only emails.
The message field accepts up to 500 chars.
All fields have placeholder text.
All fields have the correct type.
A user cannot submit the form if one or more of the fields is empty.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -283,9 +283,9 @@ <h2 class="lang-h2">Skills</h2>
         <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
             please get in touch.</h2>
         <form class="form">
```
Comment (AmaduKamara) [inline index.html]: - [ ] Kindly add the action and method attribute to the form with the action attribute's value having your [Form Spree](https://formspree.io/html/) URL you created on Form Spree and the method is set to POST that allows for posting form data to that Form Spree URL so messages can be sent to your email. Please work on that 👍 
- [ ] Kindly click on the link above, go to the Form Spree website, create your account, then create a form and take the URL of the form created and pass it to your form's action attribute. 👍

File: index.html [hunk 2]
```diff
@@ -283,9 +283,9 @@ <h2 class="lang-h2">Skills</h2>
...
-            <input class="fullname" type="text" name="fullname" placeholder="Full name">
-            <input class="fullname" type="text" name="fullname" placeholder="Email address">
-            <textarea class="text-area" placeholder="Enter text here"></textarea>
+            <input class="fullname" type="text" name="fullname" placeholder="Full name" maxlength="30" required >
+            <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
+            <textarea class="text-area" placeholder="Enter text here" maxlength="500"></textarea>
```
Comment (AmaduKamara) [inline index.html]: - [ ] Kindly add the required attribute to the `textarea` so validation is set on it and users must be able to write you a message before the form can be set.

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @zakiyousufi,

AlonzoLeroneThisCloseGIF (2)

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct use of Github workflow
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 3]
```diff
@@ -282,10 +282,10 @@ <h2 class="lang-h2">Skills</h2>
...
         <h2 class="lastpage-heading">I'm always interested in hearing about new projects, so if you'd like to chat
             please get in touch.</h2>
-        <form class="form">
-            <input class="fullname" type="text" name="fullname" placeholder="Full name">
-            <input class="fullname" type="text" name="fullname" placeholder="Email address">
-            <textarea class="text-area" placeholder="Enter text here"></textarea>
+        <form class="form" action="https://formspree.io/f/xnqwwrzy" method="POST">
+            <input class="fullname" type="text" name="fullname" placeholder="Full name" required>
+            <input class="fullname" type="email" name="fullname" placeholder="Email address" required>
+            <textarea class="text-area" placeholder="Enter text here" required></textarea>
```
Comment (uwadonat) [inline index.html:288]: - [ ] Use `message` input type on this textarea.

**Review by uwadonat — APPROVED** [review]
Hi @zakiyousufi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
appr2

# Status: Approved ✔️✔️✔️

## Optional suggestions
_check inline comments_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 3 above)
Comment (zakiyousufi) [reply index.html:288]: Thank you so mach

**Pull request merged**